x-pack/plugins/xpack_main/server/lib/telemetry/monitoring/get_high_level_stats.js

```diff
@@ -175,20 +175,34 @@ export function getHighLevelStats(server, callCluster, clusterUuids, start, end,
     .then(response => handleHighLevelStatsResponse(response, product));
 }
 
-/**
- * Fetch the high level stats to report for the {@code product}.
- *
- * @param {Object} server The server instance
- * @param {function} callCluster The callWithRequest or callWithInternalUser handler
- * @param {Array} indices The indices to use for the request
- * @param {Array} clusterUuids Cluster UUIDs to limit the request against
- * @param {Date} start Start time to limit the stats
- * @param {Date} end End time to limit the stats
- * @param {String} product The product to limit too ('kibana', 'logstash', 'beats')
- * @return {Promise} Response for the instances to fetch detailed for the product.
- */
-export function fetchHighLevelStats(server, callCluster, clusterUuids, start, end, product) {
+export async function fetchHighLevelStats(server, callCluster, clusterUuids, start, end, product) {
   const config = server.config();
+  const isKibanaIndex = product === KIBANA_SYSTEM_ID;
+  const filters = [
+    { terms: { cluster_uuid: clusterUuids } },
+  ];
+
+  // supply this from
```

Comment:
I assume this comment is meant to say something like

```js
// we should supply this from a parameter in the future so that this remains generic
```